apps/etterlatte-saksbehandling-ui/client/src/shared/TekstMedBeholdtWhitespace.tsx

```diff
@@ -0,0 +1,14 @@
+import { BodyShort, Box } from '@navikt/ds-react'
+import React from 'react'
+import { BodyShortProps } from '@navikt/ds-react/src/typography/BodyShort'
+import { OverridableComponent } from '@navikt/ds-react/src/util/types'
+
+export const TekstMedBeholdtWhitespace: OverridableComponent<BodyShortProps, HTMLParagraphElement> = (
```

Comment:
Olala, denne kan settes på flere steder om du lar folk sende med width prop

Reply:
Jeg tenker 42.5rem er en sane maxbredde på tekst, siden det ideelt ikke skal være bredere enn dette for noe som skal leses. Hvis man trenger egen maxbredde mindre enn dette kan man bare wrappe i en egen container som setter bredden